src/main/java/org/apache/accumulo/testing/continuous/BulkIngest.java

```diff
@@ -76,7 +78,8 @@ public int run(String[] args) throws Exception {
 
       // output RFiles for the import
       job.setOutputFormatClass(AccumuloFileOutputFormat.class);
-      AccumuloFileOutputFormat.configure().outputPath(new Path(bulkDir + "/files")).store(job);
+      AccumuloFileOutputFormat.configure().outputPath(new Path(fs.getUri() + bulkDir + "/files"))
```

Comment:
With the following, may not need `bulkDir`.  I think the `makeQualified` fuction only prefixes the fs URI if its not there.

```suggestion
      AccumuloFileOutputFormat.configure().outputPath(fs.makeQualified(new Path(bulkPath + "/files")))
```

Reply:
I will modify and check

Reply:
I have changed the code to use fs.makeQualified consistently and reuse existing bulkDir variable avoiding new variable bulkPath